Signal/src/ViewControllers/MessageComposeTableViewController.m

```diff
@@ -101,6 +115,20 @@ - (void)loadView
     [self updateTableContents];
 }
 
+- (void)showContactsPermissionReminder:(BOOL)isVisible
+{
+    _hideContactsPermissionReminderViewConstraint.active = !isVisible;
+}
+
+- (void)showSearchBar:(BOOL)isVisible
+{
+    if (isVisible) {
+        self.tableViewController.tableView.tableHeaderView = self.searchBar;
+    } else {
+        self.tableViewController.tableView.tableHeaderView = nil;
```

Comment:
Good point. I think ultimately the find by phone number screen should incorporate the faster feedback and "invite" functionality currently available on the search bar. 

I think in the immediate I'll replace the search bar and add a card.